### PR TITLE
Fix command history rendering for long input and update UG for command history details

### PR DESCRIPTION
**Pull request by YowSiaoKang**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -303,14 +308,11 @@ Exports data to a comma-separated values (CSV) file located at `[JAR file locati
 
 **:information_source: Notes about `export` command:**<br>
 
-* When using the application for the first time, executing the `export` command when the `addressbook.json` is missing will result in an error. Try executing other commands first. This will result `addressbook.json` file to be created.
-* When the `perons.csv` or `assignments.csv` files are being used by another application running `export` command will result in an error.
+- When using the application for the first time, executing the `export` command when the `addressbook.json` is missing will result in an error. Try executing other commands first. This will result `addressbook.json` file to be created.
+- When the `perons.csv` or `assignments.csv` files are being used by another application running `export` command will result in an error.
```
Comment (SimKianSeng) [inline docs/UserGuide.md:312]: seems like there is a typo here, should be `persons.csv`

**Review by SimKianSeng — CHANGES_REQUESTED** [review]
Minor spelling errors. Otherwise ready to merge 👍🏻 .

File: docs/UserGuide.md (hunk 1 above)
Comment (YowSiaoKang) [reply docs/UserGuide.md]: Updated

**Review by SimKianSeng — APPROVED** [review]
LGTM 👍🏻

**Review by xuwen-ho — APPROVED** [review]
well done

**Pull request merged**